test/metabase/api/dashboard_test.clj

```diff
@@ -2567,10 +2567,6 @@
             (let [execute-path (format "dashboard/%s/dashcard/%s/execute"
                                        dashboard-id
                                        dashcard-id)]
-              (testing "Without actions enabled"
-                (is (= "Actions are not enabled."
-                       (mt/user-http-request :crowberto :post 400 execute-path
-                                             {:parameters {"name" "Birds"}}))))
```

Comment:
We still need to test this fails if actions are not enabled on the database. I think this is a missing test from before actually, since it should have been tested separately from the global feature flag.